suite-common/wallet-core/src/fiat-rates/fiatRatesConstants.ts

```diff
@@ -6,11 +6,11 @@ const ONE_MINUTE_IN_MS = 60 * 1000;
 const ONE_HOUR_IN_MS = 60 * ONE_MINUTE_IN_MS;
 
 export const MAX_AGE = {
-    current: 10 * ONE_MINUTE_IN_MS,
+    current: 12 * ONE_MINUTE_IN_MS,
```

Comment:
Why 12? Ideally use constant and add comment why.

Reply:
Was making compromise between common and suite number. Will you be okey from mobile to keep it 10?

Reply:
Ok, let's keep it.